scripts/new_project.kts

```diff
@@ -41,6 +45,38 @@ object NewProject {
         }
     }
 
+    private fun renamePackageNameFolders() {
+        showMessage("=> 🔎 Rename the package folders...")
+        modules.forEach { module ->
+            val srcPath = projectPath + File.separator + module + File.separator + "src"
+            File(srcPath)
+                .walk()
+                .maxDepth(2)
+                .filter { it.isDirectory && it.name == "java" }
+                .forEach { javaDirectory ->
+                    val oldDirectory = File(
+                        javaDirectory, TEMPLATE_PACKAGE_NAME.replace(
+                            oldValue = DOT_SEPARATOR,
+                            newValue = File.separator
+                        )
+                    )
+                    val newDirectory = File(
+                        javaDirectory, packageName.replace(
+                            oldValue = DOT_SEPARATOR,
+                            newValue = File.separator
+                        )
+                    )
+
+                    val tempDirectory = File(javaDirectory, "temp_directory")
+                    oldDirectory.copyRecursively(tempDirectory)
+                    oldDirectory.parentFile?.parentFile?.deleteRecursively()
+                    newDirectory.mkdirs()
+                    tempDirectory.copyRecursively(newDirectory)
+                    tempDirectory.deleteRecursively()
```

Comment:
Should we handle error when try to `deleteRecursively` I just saw we handle error in `copyFiles` 🙏

Reply:
I can't think of any case where it would be an error unless `tempDirectory` doesn't exist.

But here we can be sure that `tempDirectory` as we created it ourselves, 

So I don't think we need it, what do you think? 🙏

Reply:
Ah, I see 👍  No need then 🎉